app/src/contexts/configContext.tsx

```diff
@@ -42,12 +44,17 @@ const getLocalConfig = (): IConfig => {
   const API_HOST = process.env.REACT_APP_API_HOST;
   const API_PORT = process.env.REACT_APP_API_PORT;
 
+  const CB_API_HOST = process.env.REACT_APP_CRITTERBASE_API_HOST;
+  const CB_API_PORT = process.env.REACT_APP_CRITTERBASE_API_PORT;
+
   const API_URL = (API_PORT && `${API_HOST}:${API_PORT}`) || API_HOST || 'localhost';
+  const CB_API_URL = (CB_API_PORT && `${CB_API_HOST}:${CB_API_PORT}`) || CB_API_HOST || 'localhost';
 
   const OBJECT_STORE_URL = process.env.OBJECT_STORE_URL || 'nrs.objectstore.gov.bc.ca';
   const OBJECT_STORE_BUCKET_NAME = process.env.OBJECT_STORE_BUCKET_NAME || 'gblhvt';
   return {
     API_HOST: ensureProtocol(API_URL, 'http://'),
+    CB_API_HOST: CB_API_URL,
```

Comment:
Can we wrap this in ensureProtocol as well, just to keep it consistent

Reply:
Fixed